update-lb4-docs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+ORG=strongloop
+REPO=loopback-next
+DOCS=./pages/en/lb4
+BRANCH=master
+ARCHIVE_NAME="loopback-next-$BRANCH"
+
+mkdir -p $DOCS
+# pull down all the markdown files (reserving dir structure)
+# given the GitHub ORG, REPO, and BRANCH
+#curl -L https://github.com/$ORG/$REPO/archive/$BRANCH.tar.gz | tar --strip-components 1 -C "$DOCS" --wildcards \
+# -xzvf- "$ARCHIVE_NAME/docs/*.md" "$ARCHIVE_NAME/packages/*.md"
+# "$ARCHIVE_NAME/docs/*.png"
+npm i @loopback/docs --prefix $DOCS
+mv $DOCS/node_modules/@loopback/docs $DOCS/monorepo-docs
```

Comment:
Is `/monorepo-docs` needed? Or can we just move the docs fo `$DOCS`?

Reply:
I was just using it as a placeholder, before we implement changes that override the current doc contents, but yeah it should be `site/*`.